lnutils/log.go

```diff
@@ -36,3 +38,13 @@ func NewSeparatorClosure() LogClosure {
 		return strings.Repeat("=", 80)
 	}
 }
+
+// LogPubKey returns a slog attribute for logging a public key in hex format.
+func LogPubKey(key string, pubKey *btcec.PublicKey) btclog.Attr {
```

Comment:
think you want `slog.Attr` as the return type here.  `btclog.Attr` is not an existing type.